semana_06/01_exercicio/hello.js

```diff
@@ -0,0 +1,7 @@
+// Exercicio 1
+
+//Meu primeiro script Node.js
+
+console.log("Hello, Node.js!");
```

Comment:
Poderia mudar para Hello World!
```suggestion
console.log("Hello, World!");
```